sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/theme/Color.kt

```diff
@@ -1,9 +1,7 @@
+@file:Suppress("MagicNumber")
```

Comment:
Why not update `MagicNumber` in `detekt-rules.yml` instead of suppressing this? 🤔 
```
  MagicNumber:
    ignorePropertyDeclaration: true
```

Reply:
@ryan-conway After discussing with @Wadeewee, we worry that setting this rule will also affect other non-composable functions. For example, we will not get any warning if we use a magic number in an area unrelated to compose. Moreover, there should be only two files we have to use the magic numbers: Color and Dimension. What do you think?

Reply:
That makes sense, but we should reconsider in case we end need to use magic number somewhere else. Sounds good to me! 👍🏻

Reply:
Agree, thank you